project/src/main/webapp/events-feed.js

```diff
@@ -1,15 +1,20 @@
-$(async function() {
-  const allEvents = await getAllEvents();
-  populateAllEvents(allEvents);
+$(function() {
```

Comment:
notation looks a bit odd, but I am not too familiar with jquery. Is there a reason not to use onload here?

Reply:
It's the same as document.ready, just shorthand for that. It's specific to jquery